docs/resources/Auto_Moderation.md

```diff
@@ -184,12 +184,11 @@ An action which will execute whenever a rule is triggered.
 Additional data used when an action is executed. Different fields are relevant based on the
 value of [action type](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-action-object-action-types).
 
-| Field            | Type      | Associated Action Types | Description                                    |
-| ---------------- | --------- | ----------------------- | ---------------------------------------------- |
-| channel_id       | snowflake | SEND_ALERT_MESSAGE      | channel to which user content should be logged |
-| duration_seconds | integer   | TIMEOUT                 | timeout duration in seconds *                  |
-
-\* Maximum of 2419200 seconds (4 weeks)
+| Field            | Type      | Associated Action Types | Description                                                                                                 | Constraints                          |
+| ---------------- | --------- | ----------------------- | ----------------------------------------------------------------------------------------------------------- | ------------------------------------ |
+| channel_id       | snowflake | SEND_ALERT_MESSAGE      | channel to which user content should be logged                                                              | existing channel                     |
+| duration_seconds | integer   | TIMEOUT                 | timeout duration in seconds                                                                                 | Maximum of 2419200 seconds (4 weeks) |
+| custom_message?  | string    | BLOCK_MESSAGE           | additional explanation that will be shown to members whenever their message is blocked (Max 150 characters) | Maximum of 150 characters            |
```

Comment:
(Max 150 characters) | Maximum of 150 characters
is it needed as duplicated info?

Reply:
oops good catch, nope that was before i made the other column

Reply:
haha, i gotchu